applications/EmpireApplication/tests/fsi_potential_flow_sdof/naca0012_mesh_movement_Y_ref.dat

```diff
@@ -0,0 +1,3 @@
+# Results for "node" with Id # 9 at position: x: 0.49032; y: -0.00891; z: 0
+# time MESH_DISPLACEMENT_Y
+0.0 5.086125656643907e-06
```

Comment:
If you want to check doubles, you can do that at the python level ([see CompFlow tests](https://github.com/KratosMultiphysics/Kratos/blob/230c3a9ec26e82cc2c5674aca4db53a89594cdb4/applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py#L63))